### PR TITLE
case reporting bits

### PR DESCRIPTION
**Pull request by snopoke**
@benrudolph @NoahCarnahan

### CODE REVIEW

File: corehq/toggles.py [hunk 1]
```diff
@@ -694,13 +694,12 @@ def _commtrackify(domain_name, toggle_is_enabled):
...
 )
 
-# Removed until ready for production
-# USE_SQL_BACKEND = StaticToggle(
-#     'sql_backend',
-#     'Uses a sql backend instead of a couch backend for form processing',
-#     TAG_PRODUCT_CORE,
-#     [NAMESPACE_DOMAIN]
-# )
+USE_SQL_BACKEND = StaticToggle(
```
Comment (benrudolph) [inline corehq/toggles.py]: :smiling_imp:

File: settings.py [hunk 2]
```diff
@@ -1612,3 +1612,7 @@
     pass
 else:
     initialize(DATADOG_API_KEY, DATADOG_APP_KEY)
+
+REST_FRAMEWORK = {
```
Comment (benrudolph) [inline settings.py]: what is this used for?

Comment (snopoke) [inline settings.py]: We use the Django Rest Framework for serializing the SQL forms and cases.
This sets the default date format.

Comment (benrudolph) [inline settings.py]: right. thanks, didn't realize this affected that

File: corehq/toggles.py (hunk 1 above)
Comment (czue) [inline corehq/toggles.py]: and so begins the journey... :rowboat:

**Pull request merged**